en/software_update/stm32_bootloader.md

```diff
@@ -60,16 +59,20 @@ These instructions are for the [J-Link GDB server](https://www.segger.com/jlink-
 
 #### Run the JLink GDB server
 
-FMUv1:
-```bash
-JLinkGDBServer -select USB=0 -device STM32F405RG -if SWD-DP -speed 20000
-```
+The command below is used to run the server for autopilots that use the STM32F427VI SoC:
 
-AeroFC:
 ```bash
-JLinkGDBServer -select USB=0 -device STM32F429AI -if SWD-DP -speed 20000
+JLinkGDBServer -select USB=0 -device STM32F427VI -if SWD-DP -speed 20000
 ```
 
+The `--device`/SoC for common targets is:
+
+* **FMUv2, FMUv3, FMUv4, aerofc-v1, mindpx-v2:** STM32F427VI
+* **px4fmu-v4pro:** STM32F469I
```

Comment:
@davids5  As per https://github.com/PX4/Firmware/issues/7758#issuecomment-321431521 you're right, aero was wrong SoC. I've updated as you suggested. I copied the name from the  CONFIG_ARCH_CHIP_ variable for the remainder of the boards - can you confirm they are OK? (specifically if config is CONFIG_ARCH_CHIP_STM32F765II , then is STM32F765II the SOC?)